### PR TITLE
Add cli test usage

### PR DESCRIPTION
**Pull request by HawieLong**
Add Python cli convenient usage example and test

Signed-off-by: Long Hanyu <hanyu.long@intel.com>

### CODE REVIEW

File: test/cli/generate_quote.py [hunk 1]
```diff
@@ -20,17 +20,28 @@ def get_args():
...
+    print('generate_quote req:\n%s\n' %(params))
+    GenerateQuote_resp = requests.post(url=base_url + "GenerateQuote", data=json.dumps(params), headers=_utils_.headers)
+    if(_utils_.check_result(GenerateQuote_resp, 'GenerateQuote') == False):
+        return
+    print('generate_quote resp:\n%s\n' %(GenerateQuote_resp.text))
+
+    f = open(quote_file, "w")
     f.write(json.loads(GenerateQuote_resp.text)['result']['quote'])
 
 if __name__ == "__main__":
```
Comment (syan10) [inline test/cli/generate_quote.py:47]: does it block the cli script?

Comment (HawieLong) [reply test/cli/generate_quote.py:47]: no, function name used in main is changed

File: test/test_kms_with_cli.py [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+
+    verify_quote.verify_quote_with_file(base_url, "a.txt")
+
+    print('====================test_GenerateQuote_and_VerifyQuote end===========================')
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-i', '--ip-adress', type=str, help='ip address of the ehsm_kms_server', required=True)
+    parser.add_argument('-p', '--port', type=str, help='port of the ehsm_kms_server', required=True)
```
Comment (syan10) [inline test/test_kms_with_cli.py:152]: please refine it, and align it same with the cli scripts, e.g
test.py --url http://1.2.3.4:9000/

File: test/test_kms_with_cli.py [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+    args = parser.parse_args()
+    ip = args.ip_adress
+    port = args.port
+    return ip, port
+    
+def check_result(res, action, test_name):
+    res_json = json.loads(res.text)
```
Comment (syan10) [inline test/test_kms_with_cli.py:159]: does it still need?

File: test/_utils_.py [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  return {k: str(v).casefold() if isinstance(v, bool) else v for k, v in pairs}
+
+def check_result(res, action):
+    res_json = json.loads(res.text)
+    if(res_json['code'] == 200):
+        print("%s successfully \n" %(action))
+        return True
+    else:
+        print("%s failed, error message: %s \n" %(action, res_json["message"]))
+        return False
```
Comment (syan10) [inline test/_utils_.py:46]: why add a empty file "cli/__init__.py"

Comment (HawieLong) [reply test/_utils_.py:46]: to initiate the cli folder as a import-able folder

File: test/test_kms_with_cli.py (hunk 3 above)
Comment (HawieLong) [reply test/test_kms_with_cli.py:159]: not need any more, deleted

File: test/test_kms_with_cli.py (hunk 2 above)
Comment (HawieLong) [reply test/test_kms_with_cli.py]: fixed

**Review by syan10 — APPROVED** [review]

**Pull request merged**